gms/impl/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
   compile externalDependency.gmaNeo4jDao
   compile externalDependency.gmaRestliResources
   compile externalDependency.gmaRestliResourcesDataTemplate
+  compile externalDependency.neo4jJavaDriver
```

Comment:
Just curious - why wasn't this necessary for prior versions?

Reply:
+1 on this question

Reply:
Good question!

I must have been made a mistake when I first compiled the project, since now it does get included in the WAR 

The driver is brought in from `metadata-utils`:
`compile project(':metadata-utils')`. 
https://github.com/linkedin/datahub/blob/master/metadata-jobs/mae-consumer-job/build.gradle#L19

However, since datahub-mae has an [explicit dependency](https://github.com/linkedin/datahub/blob/master/metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/config/Neo4jConfig.java#L3) to the driver it might make sense to declare it explicitly in `build.gradle` as well. What do you think?

Reply:
Thanks! That makes sense.